example-application/libraries/mailer.js

```diff
@@ -1,3 +1,6 @@
-module.exports.send = (subject, body, recipientAddress) => {
+const axios = require('axios');
+
+module.exports.send = async (subject, body, recipientAddress) => {
   console.log('Not really a mailer, right?', subject, body, recipientAddress);
+  await axios.post(`http://localhost/mailer`, { subject, body, recipientAddress });
```

Comment:
See the comment 'Intercept vs stub comment' below

We have to decide whether mailer is our code or npm lib. If the later, we should not intercept the URL